src/components/PrefixedEthHashInfo/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import PrefixedEthHashInfo from '.'
+import { render, screen } from 'src/utils/test-utils'
+
+const hash = '0x69904ff6d6100799344E5C9A2806936318F6ba4f'
+jest.mock('src/routes/routes', () => {
+  const original = jest.requireActual('src/routes/routes')
+  return {
+    ...original,
+    extractShortChainName: () => 'arb',
+  }
+})
+
+describe('<PrefixedEthHashInfo>', () => {
```

Comment:
Can you not also add a test to check that copying also works?

Reply:
The copy functionality is abstracted by the `SRC` components so I am not sure how to test it... I am trying to assert the clipboard content but I don't think we should be testing that in `safe-react`. WDYT?

Reply:
(_Answering the comment_) I think I have covered all the address visualizations in the UI. The ones I haven't prefixed I comment in https://github.com/gnosis/safe-react/pull/2896#issuecomment-954622380 . Said this, I will ask QA to help me spotting any edge case that I might have missed.

Reply:
You could still render the component, click copy and see what is in the clipboard, no? Did you test it in SRC?